.cljstyle

```diff
@@ -0,0 +1,29 @@
+{:rules
```

Comment:
This style makes things hard to read:
```
(my-function param
   another-param)
```
this is much easier:
```
(my-function param
                       another-param)
;;or
(my-function
  param
  another-param)
```
The defaults of cljstyle without a .cljstyle work quite well I think?

Reply:
also i think a lot of indentation updates are because of this rule, is that correct?